mipengine/controller/node_registry.py

```diff
@@ -13,6 +13,9 @@
 from mipengine.node_info_DTOs import NodeInfo
 from mipengine.node_info_DTOs import NodeRole
 
+# TODO remove import get_node_celery_app, pass the celery app  (inverse dependency)
```

Comment:
The node_registry is unit tested here https://github.com/madgik/MIP-Engine/blob/dev/MIP-184_separate-controller-algorithmexecutor/tests/unit_tests/test_node_registry.py , maybe tests for get_all_available_schemas and get_all_available_datasets_per_schema should be added.

Reply:
https://team-1617704806227.atlassian.net/browse/MIP-255